src/main/java/edu/kis/powp/jobs2d/drivers/DrawLineManager.java

```diff
@@ -0,0 +1,27 @@
+package edu.kis.powp.jobs2d.drivers;
+
+import edu.kis.legacy.drawer.shape.ILine;
+import edu.kis.powp.jobs2d.AbstractDriver;
+import edu.kis.legacy.drawer.shape.LineFactory;
+import edu.kis.powp.jobs2d.features.DrawerFeature;
+
+public class DrawLineManager extends AbstractDriver {
```

Comment:
Why manager?

Reply:
Changed it to adapter as it adapts AbstractDriver into that project.